### PR TITLE
Bugs fixed. Texts fixed. Re-coded as responsive.

### PR DESCRIPTION
**Pull request by aligunesv**
Fixed some bugs (not using tags like p and h1 h2). Missing codes fixed. Unnecessary codes have been deleted. Made mobile compatible.

### CODE REVIEW

File: README.md
Comment (aligunesv) [inline README.md:1]: I added my own link to the contributors.

File: index.html
Comment (aligunesv) [inline index.html:1]: fixed errors and unnecessary usage

File: me.css
Comment (aligunesv) [inline me.css:1]: I removed the irregular spelling and made it more neatly understandable. I made it mobile friendly.

**Review by aligunesv — COMMENTED** [review]
I removed the irregular spelling and made it more neatly understandable. I made it mobile friendly.

**Comment by Manvi-Chaudhary** [comment]
> Fixed some bugs (not using tags like p and h1 h2). Missing codes fixed. Unnecessary codes have been deleted. Made mobile compatible.

I am thankful for your contribution. However, i think you have changed the font style of website. My website is more mobile responsive now than before, but still it is not compatible in mobile as tags are overlapping each other. I wonder if you can fix this issue.
image

**Review by aligunesv — COMMENTED** [review]
i fixed responsive design.

**Review by Manvi-Chaudhary — APPROVED** [review]

**Pull request merged**